source/opt/inline_pass.cpp

```diff
@@ -228,9 +228,9 @@ void InlinePass::GenInlineCode(
   ir::Function* calleeFn = id2function_[call_inst_itr->GetSingleWordOperand(
       kSpvFunctionCallFunctionId)];
 
-  // Check for early returns
-  auto fi = early_return_.find(calleeFn->result_id());
-  bool earlyReturn = fi != early_return_.end();
+  // Check for multiple returns in the callee.
+  auto fi = multi_return_funcs_.find(calleeFn->result_id());
+  bool multiReturn = fi != multi_return_funcs_.end();
```

Comment:
Fixed in next round.  I also updated the lambda to capture multiReturn by value rather than reference.